src/main/java/com/example/demo/services/ReceiptService.java

```diff
@@ -0,0 +1,59 @@
+package com.example.demo.services;
+import com.example.demo.database.Receipt;
+import com.example.demo.repository.ReceiptRepository;
+import org.springframework.data.crossstore.ChangeSetPersister;
+import org.springframework.stereotype.Service;
+import java.util.Date;
+import java.util.List;
+
+
+@Service
+public class ReceiptService {
+    private final ReceiptRepository receiptRepository;
+
+    public ReceiptService(ReceiptRepository receiptRepository) {
+        this.receiptRepository = receiptRepository;
+    }
+
+    public Receipt createReceipt(Receipt receipt) {
+        // Логіка створення чеку
```

Comment:
1. There is no reason to put comments here.
2. At least comments should be in English